test/Unit/lit.cfg

```diff
@@ -63,7 +63,7 @@ elif swift_test_subset == 'only_stress':
     #   Currently those tests are very fast so it doesn't matter much.
     pass
 else:
-    lit_config.fatal("Unknown test mode %r" % swift_test_subset)
+    lit_config.fatal("Unknown test subset %r" % swift_test_subset)
```

Comment:
What about if we run long tests... why not run long tests with early swift driver?

Reply:
The intent is to have a small set of tests that verify the early driver's basic functionality. 

When running compiler's test suites (such as the long tests), we run them with the legacy driver, because that too needs validation. But most importantly, when building and testing the actual SwiftDriver build product (not early), it has a setup where it will run driver-relevant lit tests https://github.com/apple/swift-driver/blob/main/Tests/SwiftDriverTests/IntegrationTests.swift#L103. This is where we should actually thoroughly test things, and consider running the long tests. 

There are two separate directions for the future:
- Reduce the lit tests that exercise the legacy driver over time to verify only the most basic functionality, transitioning them to instead run when testing the new driver.
- Gradually port over driver-specific tests to the driver repository's XCTests, instead of lit.

Reply:
Ok. I am fine with this. I was just curious if this was an over sight.